scripts/core/exp-command-buffer.rst

```diff
@@ -0,0 +1,116 @@
+<%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+.. _experimental-command-buffer:
+
+==============================
+Command-Buffer
+==============================
+
+.. warning::
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions
+        over time.
+    *   Do not require conformance testing of their own additions.
+
+
+A command-buffer represents a series of commands for execution on a command queue.
+Many adapters support this kind of construct either natively or through extensions,
+but they are not available to use directly. Typically their use is abstracted through
+the existing Core APIs, for example when calling ${x}EnqueueKernelLaunch the adapter
+may both append the kernel command to a command-buffer-like construct and also submit
+that command-buffer to a queue for execution. These types of structures allow for
+batching of commands to improve host launch latency, but without direct control it
+falls to the adapter implementation to implement automatic batching of commands.
+
+This experimental feature exposes command-buffers in the Unified Runtime API directly,
+allowing applications explicit control over the enqueue and execution of commands to
+batch commands as required for optimal performance.
+
+Querying Command-Buffer Support
+================================
```

Comment:
nitpick - think some of these section underscores are a character or two longer than the section title

Reply:
Fixed, thanks!